src/date-range-picker/calendar/__tests__/utils.test.ts

```diff
@@ -0,0 +1,188 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { findDateToFocus, findMonthToDisplay, findMonthToFocus, findYearToDisplay } from '../utils';
+// Helper function to create Date objects from YYYY-MM-DD strings
+const createDate = (dateString: string) => new Date(dateString);
+describe('findDateToFocus', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    jest.useFakeTimers();
+    jest.setSystemTime(createDate('2023-06-15').getTime()); // Set a fixed date for 'today'
```

Comment:
This is also interesting, how is that different than the mockdate library and should we have some consistency in dates mocking?